test/src/test/java/com/distkv/client/commandlinetool/DistkvCommandExecutorHandlerTest.java

```diff
@@ -242,26 +242,24 @@ public void testMain() {
       command = "int.get k1";
       distKVParsedResult = distkvParser.parse(command);
       Assert.assertEquals(
-          CommandExecutorHandler.intGet(distkvClient, distKVParsedResult), 12);
-
-      // Test command 'int.drop'
-      command = "int.drop k1";
-      distKVParsedResult = distkvParser.parse(command);
-      Assert.assertEquals(
-          CommandExecutorHandler.intDrop(distkvClient, distKVParsedResult), STATUS_OK);
+          CommandExecutorHandler.intGet(distkvClient, distKVParsedResult), String.valueOf(12));
 
       // Test command 'int.incr'
-      command = "int.put k2 10";
-      distKVParsedResult = distkvParser.parse(command);
-      command = "int.incr k2 -3";
+      command = "int.incr k1 -3";
       distKVParsedResult = distkvParser.parse(command);
       Assert.assertEquals(
           CommandExecutorHandler.intIncr(distkvClient, distKVParsedResult), STATUS_OK);
```

Comment:
It's better to check the value of k1 after increment -3 is excepted.

Reply:
OK, I see